src/lucky/exceptions.cr

```diff
@@ -22,6 +23,10 @@ module Lucky
       def message : String?
         "Required param \"#{param_name}\" with value \"#{param_value}\" couldn't be parsed to a \"#{param_type}\""
       end
+
+      def default_response_status_code
+        422
```

Comment:
Maybe this could use one of the Staus enums so future readers don’t have to look it up?